.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[Bug] : "
+labels: bug, triage
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+<!-- **Please add a reproducible repo or sandbox** -->
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem. (from [cssnano-playground](https://cssnano.co/playground/))
+
+**Desktop (please complete the following information):**
+- CSSNANO Version [e.g. 22] 
+-  Plugin version (if it's known)
+<!-- also mention the version in which it was version (if any) -->
```

Comment:
Will be great to recommend tool like `env` + `npm ls cssnano` to collect all information

Reply:
any suggestion for tool which gives that without like without  adding it to the cli?


> `npm ls cssnano`

Ok cool

Reply:
`envinfo` good tool

Reply:
ok cool .. I will add `npx envinfo` in it and `npm ls cssnano` as well

Reply:
Yep

Reply:
done